app/utils/autoscale.ts

```diff
@@ -0,0 +1,40 @@
+import * as CodeMirror from "codemirror";
+
+const variables = [
+    "ActiveTasks", "CPUPercent", "CurrentDedicated", "DiskBytes",
+    "DiskReadBytes", "DiskReadOps", "DiskWriteBytes", "DiskWriteOps",
+    "FailedTasks", "MemoryBytes", "NetworkInBytes", "NetworkOutBytes",
+    "PendingTasks", "RunningTasks", "SampleNodeCount", "SucceededTasks",
+    "TargetDedicated", "$WallClockSeconds",
+];
+
+// tslint:disable-next-line:max-line-length
+const keywordRegex = /\$CPUPercent|\$WallClockSeconds|\$MemoryBytes|\$DiskBytes|\$DiskReadBytes\$DiskWriteBytes|\$DiskReadOps|\$DiskWriteOps|\$NetworkInBytes|\$NetworkOutBytes|\$SampleNodeCount|\$ActiveTasks|\$RunningTasks|\$PendingTasks|\$SucceededTasks|\$FailedTasks|\$CurrentDedicated|\$TargetDedicated/;
```

Comment:
I don't know how the codemirror modes works but could you here build the regex from an array here instead. It would look cleaner I think.